<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>900</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color : qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(223, 95, 159, 200), stop:0.513812 rgba(224, 212, 107, 200), stop:1 rgba(103, 188, 228, 200))</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>300</y>
     <width>400</width>
     <height>300</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);

border-radius : 10px;</string>
   </property>
   <widget class="QPushButton" name="start_button">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color : rgb(255, 255, 255);
background-color : qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(223, 185, 204, 255), stop:0.510989 rgba(224, 219, 177, 255), stop:1 rgba(148, 202, 228, 255));

font-family : 'Dosis';
font-weight : bold;
font-size : 14pt;

border-radius : 16px;</string>
    </property>
    <property name="text">
     <string>Ready To Date?</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="member_num">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>200</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 54);
color: rgb(255, 255, 255);

border-radius : 8px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family : 'Dosis';
font-size : 12pt;</string>
    </property>
    <property name="text">
     <string>How many?</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>240</width>
      <height>120</height>
     </rect>
    </property>
    <widget class="QLabel" name="DM_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>200</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Colonna MT</family>
       <pointsize>40</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color : rgb(0, 0, 0);
background-color : rgba(255, 255, 255, 0);

font-family : 'Colonna MT';
font-weight : bold;
font-size : 40pt; </string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Manager&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="DM_label_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>0</y>
       <width>120</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Colonna MT</family>
       <pointsize>40</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color : rgb(0, 0, 0);
background-color : rgba(255, 255, 255, 0);

font-family : 'Colonna MT';
font-weight : bold;
font-size : 40pt; </string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Date&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>start_clicked()</slot>
 </slots>
</ui>
